android/build.gradle

```diff
@@ -26,6 +26,7 @@ buildscript {
   ext.googleFirebaseServicesEnabled = project.hasProperty('firebase') ?: googleFirebaseServicesDefault
 
   dependencies {
+    classpath 'com.android.tools:r8:3.3.28'
```

Comment:
is it necessary to workaround the bug?
otherwise its better to use default R8 version included in AGP I suppose

Reply:
There is another bug without this explicit version, that was fixed by google.